src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/service/HdcService.kt

```diff
@@ -60,6 +60,25 @@ class HdcService(
     )
   }
 
```

Comment:
Combined the checks into a function - placed it in the `HdcService` but happy to move it if it is not the best place?